### DISCUSSION TITLE
Spring boot rest Microservice Authentication

### DISCUSSION THREAD

**Discussion by jkumar0021** [javax4u/interview-questions#38] [Q&A]
Why we are using the GWT or Oath2 token in spring boot rest microservice?
Can we use LDAP Authentication in spring boot rest microservice?
Which Authentication is better for rest services?

**Comment by javax4u** [comment]
![Technology Track Discussion (eug-guwa-jcz – 5 Nov 2021) 1](https://user-images.githubusercontent.com/2442530/140502507-90d7bb13-a0a3-4c72-a53c-2de2fc24150b.png)
